src/accelerate/big_modeling.py

```diff
@@ -47,13 +48,13 @@
 
 
 @contextmanager
-def init_empty_weights(include_buffers: bool = False):
+def init_empty_weights(include_buffers: bool = None):
     """
     A context manager under which models are initialized with all parameters on the meta device, therefore creating an
     empty model. Useful when just initializing the model would blow the available RAM.
 
     Args:
-        include_buffers (`bool`, *optional*, defaults to `False`):
+        include_buffers (`bool`, *optional*, defaults to `None`):
```

Comment:
```suggestion
        include_buffers (`bool`, *optional*):
```
Small nit: we don't put the default when it's None as it's implied by `*optional*`